driver/makedepend.ml

```diff
@@ -574,10 +574,11 @@ let print_version_num () =
 
 let main () =
   Clflags.classic := false;
-  add_to_list first_include_dirs Filename.current_dir_name;
   Compenv.readenv ppf Before_args;
   Clflags.reset_arguments (); (* reset arguments from ocamlc/ocamlopt *)
   Clflags.add_arguments __LOC__ [
+     "-exclude-cur-dir", Arg.Set Clflags.exclude_dir,
```

Comment:
The name `-exclude-cur-dir` is a bit too ambiguous. Which cur is excluded from the directory? What did the poor dog do? More seriously, It is better to try to convey the intention behind the argument. Here, we are dealing with include directories, and we want to make them more explicit. The name of the argument should try to convey that.

There is no variable called `exlude_dir` in Clflags. It is better for now to define a local reference.

Reply:
would a better name be `-prune-root-dir` ? a similiar thing is used in `find` command in linux.
would the local variable name be also be better as `prune_root_dir` for consistency?

Reply:
As an argument `-prune-root-dir` does not seem to fit well because the object of ocamldep is not to explore directories contrarily to `find`. Here, the flag is a modifier to include directories, which are a configuration option themselves. Thus, I think it is important to keep `include` in the name. Similarly, it is indeed a goiod indeed to keep the local variable name as close as possible from the argument name.

Note that the name will probably be the last thing to be refined, so it is fine to start with a working argument name.

Reply:
My own idea for option names would be `-no-working-dir` (by analogy with the not-so-well-named `-nostdlib` flag of ocamlc, which does not add the standard library to the list of include directories), and I would have considered `-exclude-working-dir` as well, using `-exclude` as a sort of negation of `-include`. This seems shorter than `-do-not-include-the-working-directory` and clearer than `-no-working-dir-include`.

Reply:
I would go for `-nocwd`:
```
 -nocwd  Do not add current working directory to the list of include directories
 -nostdlib  Do not add default directory to the list of include directories
```

Reply:
Another possibility might be `-no-implicit-include` but the terseness of `-nocwd` is attractve.

Reply:
If you want `-no-implicit-include` then I think it should also imply `-nostdlib`.

Reply:
`-nocwd` does seem compact and self explanatory, should I proceed with it ?

Reply:
Let's go with `-nocwd` indeed.